sentry-spring/src/main/java/io/sentry/spring/webflux/SentryWebFilter.java

```diff
@@ -81,7 +81,7 @@ isTracingEnabled && shouldTraceRequest(requestHub, request)
               if (transaction != null) {
                 finishTransaction(serverWebExchange, transaction);
               }
-              requestHub.popScope();
+              requestHub.popScope(); // TODO don't
```

Comment:
`popScope` by itself here wouldn't be problematic, however combining this with e.g. setting `request` on the `requestHub` reference, leads to unexpected results where e.g. `request` isn't set on captured messages.